src/components/TrafimageMaps/TrafimageMaps.js

```diff
@@ -13,7 +13,11 @@ import { Layer } from 'mobility-toolbox-js/ol';
 import TopicLoader from '../TopicLoader';
 import { getStore } from '../../model/store';
 import { setZoom, setCenter, setMaxExtent } from '../../model/map/actions';
-import { setLanguage, setCartaroOldUrl } from '../../model/app/actions';
+import {
+  setLanguage,
+  setCartaroOldUrl,
+  setPermissionsInfos,
```

Comment:
I've wanted to say this for a long time but I didn't dare:

I find this duplicate plural "s" in a lot of variables quite confusing. The plural of "permission info" is "permission infos" and not "permissions infos". :-)

Reply:
Common french mistake in english indeed.
I change this variable, thanks :+1:

Reply:
Unnecessary 's' removed from variables